google/cloud/pubsub/internal/tracing_pull_lease_manager_test.cc

```diff
@@ -83,6 +85,39 @@ TEST(TracingPullLeaseManagerImplTest, AsyncModifyAckDeadlineSuccess) {
                          SpanNamed("test-subscription modack"))));
 }
 
+#if OPENTELEMETRY_VERSION_MAJOR >= 2 || \
+    (OPENTELEMETRY_VERSION_MAJOR == 1 && OPENTELEMETRY_VERSION_MINOR >= 13)
+TEST(TracingPullLeaseManagerImplTest, CreateRootSpan) {
+  auto span_catcher = InstallSpanCatcher();
+  auto active_span = MakeSpan("active span");
+  auto active_scope = opentelemetry::trace::Scope(active_span);
```

Comment:
Why? It is weird to create a scope then end the span on the next line of code.

Reply:
Now it is not ended, leaving as is